### ISSUE TITLE
[Feedback] Node.js - /guides/connect/nodejs/

### ISSUE THREAD

**Issue by konotorii** [readysettech/docs-old#101]
## What is the reason for your feedback?

[x] Missing the information I need

[ ] Too complicated

[ ] Out of date

[ ] Something is broken

[ ] Other

## Additional details

Zero mention of how to integrate with Prisma ORM with either an existing or new project. The [readyset.io](https://readyset.io) page mentions that it works with Prisma yet no mention on how to integrate it.

**Comment by alanarenda** [comment]
Hey @konotorii, thank you for your feedback! We're working on writing something more specific up for Prisma, I'll follow up once it's published.

**Comment by konotorii** [comment]
Awesome, thank you! Looking forward to it.

**Comment by greg-mckeon** [comment]
Hey @konotorii - I've just pushed an update that explains how to connect existing apps here: https://docs.readyset.io/guides/connect/existing-app/

Let me know if that helps!

**Comment by konotorii** [comment]
Yes it does somewhat help, only thing to complain about is that the hyperlinks aren't linked actually.
